### ISSUE TITLE
Can we have chatbot textbox with autofill paces from google map

### ISSUE THREAD

**Issue by anithats1988** [eduardotkoller/convForm#49]
I have a chatbot using ConvForm and i have a question to add address from the user. When user enter text can we auto suggest data using google API?

**Comment by eduardotkoller** [comment]
At the moment, no. How do you imagine the suggestions should appear?

**Comment by anithats1988** [comment]
When we type in textbox,  if it is for adress then it should give suggestions as we enter keysOn 10-Apr-2019 7:16 PM, Eduardo Thomas Koller <notifications@github.com> wrote:At the moment, no. How do you imagine the suggestions should appear?

**Comment by eduardotkoller** [comment]
Yeah, I got that. I mean visually, what do you think it's the best way to
show the suggestion to the user? Just as a placeholder? As options like the
select field? As a dropdown? Just to know if you are imagining the same
thing as I am.

In anyway, I will add this as an enhancement. Unfortunely I don't have
enough time to work on this right now, so I can't promise anything, sorry.

**Comment by anithats1988** [comment]
Something like we type in google map textboxesOn 10-Apr-2019 9:33 PM, Eduardo Thomas Koller <notifications@github.com> wrote:Yeah, I got that. I mean visually, what do you think it's the best way to

show the suggestion to the user? Just as a placeholder? As options like the

select field? As a dropdown? Just to know if you are imagining the same

thing as I am.



In anyway, I will add this as an enhancement. Unfortunely I don't have

enough time to work on this right now, so I can't promise anything, sorry.

**Comment by anithats1988** [comment]
HI Sir,

                Can u help me to integrate autosuggest plugin with ConvForm?

 

https://www.jqueryscript.net/form/Autocomplete-Text-Input-AutoSuggestList.html

 

Thanks and Regards
Anitha Sonu

 <https://www.linkedin.com/in/anithasonu/> https://www.linkedin.com/in/anithasonu/

M: +91 9446067955, Skype: anitha.sonu18, Hangout:  <mailto:anitha@seeroo.in> anitha@seeroo.in
SEEROO IT SOLUTIONS Pvt. Ltd. - Powerful Thoughts - Simple Solutions  <http://www.seeroo.com> www.seeroo.com

**Comment by anithats1988** [comment]
Hi Sir,

Can u help me where can I put below code if I wish to enable autocomplete?

 

setTimeout(function(){

                                   new google.maps.places.Autocomplete(

    (  $('#userInput')), {

        types: ['cities']

    });

                //new google.maps.places.Autocomplete($(this.wrapper).find('#'+this.parameters.inputIdName));

                }.bind(this),200);

 

Thanks and Regards
Anitha Sonu

 <https://www.linkedin.com/in/anithasonu/> https://www.linkedin.com/in/anithasonu/

M: +91 9446067955, Skype: anitha.sonu18, Hangout:  <mailto:anitha@seeroo.in> anitha@seeroo.in
SEEROO IT SOLUTIONS Pvt. Ltd. - Powerful Thoughts - Simple Solutions  <http://www.seeroo.com> www.seeroo.com

**Comment by anithats1988** [comment]
Hi Sir,

                Is there any option to save chatbot data and resume conversation upon page refresh?

 

Thanks,

Anitha



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by sumitpareek1992** [comment]
Hi Sir 
           Are you help me to integrate rasa chatbot api with ConvForm


Thanks and Regards
Sumit Sharma
